packages/cli/test/mocks/team.ts

```diff
@@ -16,6 +25,25 @@ export function useTeams(teamId?: string) {
 
   for (let team of teams) {
     client.scenario.get(`/teams/${team.id}`, (_req, res) => {
+      if (options.failMissingToken) {
+        res.statusCode = 403;
+        res.json({
+          message: 'The request is missing an authentication token',
+          code: 'forbidden',
+          missingToken: true,
+        });
+        return;
+      }
+
+      if (options.failNoAccess) {
+        res.statusCode = 403;
+        res.send({
+          code: 'team_unauthorized',
+          message: 'You are not authorized',
+        });
+        return;
+      }
```

Comment:
I'm not a huge fan of this logic being here because it can fall out of sync with the actual API responses. I couldn't get an integration test that would have a missing or expired token working properly.

If someone has a suggestion or wants to pair on that, let me know!